cloudflare/resource_cloudflare_custom_ssl_test.go

```diff
@@ -0,0 +1,104 @@
+package cloudflare
+
+import (
+	"fmt"
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+	"regexp"
+	"testing"
+)
+
+func TestAccCloudflareCustomSSL_Basic(t *testing.T) {
+	t.Parallel()
+	var customSSL cloudflare.ZoneCustomSSL
+	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
+	resourceName := fmt.Sprintf("cloudflare_custom_ssl.foossl")
```

Comment:
Let's use the `generateRandomResourceName()` method here to build a resource name for the entropy to denote this isn't a static resource (only used for testing) and in the future when we swap all resource names to begin with `tf-acc`, this will be one less place we need to manually update.

```suggestion
      rnd := generateRandomResourceName()
	resourceName := "cloudflare_custom_ssl." + rnd
```

Reply:
This will require a couple of flow on updates in this test but ends up being quite a bit more flexible.

Reply:
Sounds good. Updated in commit: [8fa7bea](https://github.com/terraform-providers/terraform-provider-cloudflare/pull/418/commits/853a45f466eb637b3452e082f723e5c78ac6b2c5)